src/lib/Config/ExportCredentialsResolver.php

```diff
@@ -15,34 +15,32 @@
 
 final class ExportCredentialsResolver extends CredentialsResolver
 {
-    /** @var string */
-    private $method;
-
-    /**
-     * {@inheritdoc}
-     */
     public function getCredentials(?string $siteAccess = null): ?Credentials
     {
-        $this->method = $this->configResolver->getParameter(
-            'export.authentication.method',
-            Parameters::NAMESPACE,
-            $siteAccess
-        );
+        $requiredCredentials = $this->getRequiredCredentials($siteAccess);
 
-        if ($this->method === ExportMethod::USER && !$this->hasCredentials($siteAccess)) {
+        if ($requiredCredentials['method'] === ExportMethod::USER && !$this->hasCredentials($siteAccess)) {
```

Comment:
Is the `'method'` array entry is used somewhere else? If yes, can it be converted into a `const`, so the connection between usage here and in consuming class is more visible?